ndex2/client.py

```diff
@@ -1152,6 +1748,32 @@ def delete_networks_from_networkset(self, set_id, networks, retry=5):
                     raise inst
         raise Exception("Network is locked after " + str(retry) + " retry.")
 
+    def get_sample_network(self, network_id):
+        """
+        Gets the sample network
+
+	.. code-block:: python
+	
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		network_id = 'INSERT NETWORK UUID HERE'
+		sample_network = my_ndex.get_sample_network(network_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client